.github/workflows/build.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Set up Go 1.20
```

Comment:
can you also adjust the name to accurately reference the version of Go that is being used? better yet we could change this line to simply be `Setting up Go` that way we don't have to keep making changes every time we update the Go version being used.